app/views/ladder/LadderView.js

```diff
@@ -130,13 +131,17 @@ module.exports = (LadderView = (function () {
       this.prototype.showBackground = false
 
       this.prototype.subscriptions =
-        { 'application:idle-changed': 'onIdleChanged' }
+        {
+          'application:idle-changed': 'onIdleChanged',
+          'ladder:refresh': 'updateSpectateList'
```

Comment:
do we need this? It keeps on refreshing the dropdown and whole spectate div.  It's ok if we don't refresh the dropdown automatically and require a refresh of the page

Reply:
we need. since we update rankings all the time

Reply:
I dont have an issue with heavy/light. When I'm trying to select an element in dropdown, this refresh happens and the selection disappears. Also, I think for the global too many updates and changes in rankings seem okay, I don't see so many updates in a teacher dashboard of a teacher/class clan.